tests/ovirt/requests/compute/client_tests.rb

```diff
@@ -0,0 +1,23 @@
+Shindo.tests('Fog::Compute[:ovirt] | client', ['ovirt']) do
+  before do
+    @client_mock = Object.new
+    def @client_mock.foo
+      raise OVIRT::OvirtException, 'Test'
+    end
+
+    @object_under_test = Fog::Compute::Ovirt::ExceptionWrapper.new(@client_mock)
+  end
+
+  tests('Raises the right type of exception')
+    .raises(Fog::Ovirt::Errors::OvirtClientError) do
+    @object_under_test.foo
+  end
+
+  tests('The exception test is as expected').returns(true) do
+    begin
+      @object_under_test.foo
+    rescue Fog::Ovirt::Errors::OvirtClientError => e
+      e.message == 'Ovirt client returned an error: Test'
+    end
+  end
```

Comment:
Can you also test that you get the original exception in `orig_exception`?

Reply:
sorry, my bad, here it is :)